datastore/additional_data/generator.py

```diff
@@ -36,17 +38,22 @@ def __init__(self):
         self.code_lists = CodeListSource()
         self.tsg_recipient_type = TSGRecipientTypesSource()
         self.imd_snapshot = IMDSnapshotSource()
-        # Initialise Other Sources here
+        self.grant_metadata = GrantMetadataSource()
+        # Initialise other additional data sources here
 
-    def create(self, grant, data_sources=DATA_SOURCES):
+    def create(self, grant, source, additional_data_sources=DATA_SOURCES):
         """Takes a grant's data and returns a dict of additional data"""
 
         additional_data = {}
 
-        for source in data_sources:
+        for additional_data_source in additional_data_sources:
             try:
-                getattr(self, source).update_additional_data(grant, additional_data)
+                getattr(self, additional_data_source).update_additional_data(
+                    grant, additional_data, source=source
```

Comment:
just wondering what the rationale for using kwargs `source=source` is here?

Reply:
it was in case we want to add more params to update_additional_data in future without changing every additional data source each time (but really so I didn't have to update all the tests) but I've changed this now

Reply:
it was in case we want to add more params to update_additional_data in future without changing every additional data source each time (but really so I didn't have to update all the tests) but I've changed this now